object.c

```diff
@@ -3651,15 +3651,37 @@ rb_String(VALUE val)
 
 /*
  *  call-seq:
- *     String(arg)   -> string
+ *    String(object) -> object or new_string
  *
- *  Returns <i>arg</i> as a String.
+ *  Returns an array converted from +object+.
  *
- *  First tries to call its <code>to_str</code> method, then its <code>to_s</code> method.
+ *  If +object+ is:
+ *
+ *  - A string, returns +object+.
+ *  - +nil+, returns an empty string.
```

Comment:
`nil` behavior should not be mentioned separately, since it isn't treated separately.  The behavior for `nil` is the same for non-strings (`nil.to_s # => ''`).

Reply:
Fixed.